plugins/google_search/plugin.spec.yaml

```diff
@@ -82,6 +80,7 @@ actions:
         title: URL
         type: string
         description: URL to retrieve e.g. https://www.google.com
+        example: https://example.com
```

Comment:
When you regen, it'll normally convert these examples to this one. Change it to `https://www.google.com` and be mindful of when you regenerate, as you'll probably need to manually change it back.

Reply:
Have kept this at https://example.com to remain true to the style guide but have removed the unnecessary example in the description to avoid two separate example values being displayed.